env/env.go

```diff
@@ -7,11 +7,28 @@ import (
 	"net/url"
 	"os"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/pkg/errors"
 )
 
+// Bool parses given environment variables as an int, or returns the default if the environment variable is empty/unset.
```

Comment:
s/as int/as a boolean/
Should also document what gets parsed, 1, tT, tTrRuUeE. Should we also handle `yes`?

Reply:
ah good point, will add that